Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM debian:wheezy
 MAINTAINER Steeve Morin "steeve.morin@gmail.com"
 
+#Change libc6-i386 by libc6. Future test might involve libc6-amd64
```

Comment:
I have changed libc6-i386 for libc6 to be able to compile for a 64 bits system. I was wondering if using libc6-amd64 would be better in that regard. This is the meaning of the comment. Note that it might be a plain stupid idea, and using libc6 is perfectly fine.